ddsp/training/data.py

```diff
@@ -420,19 +424,30 @@ def __init__(self, base_dir, instrument_key='tpt', split='train'):
         ['all', 'bn', 'cl', 'db', 'fl', 'hn', 'ob', 'sax', 'tba', 'tbn',
         'tpt', 'va', 'vc', 'vn'].
       split: Choices include ['train', 'test'].
+      suffix: Choices include [None, 'batched', 'unbatched'], but broadly
+        applies to any suffix adding to the file pattern.
+        When suffix is not None, will add "_suffix" to the file pattern.
+        This option is used in gs://magentadata/datasets/urmp/urmp_20210324.
+        With the "batched" suffix, the dataloader will load tfrecords
+        containing segmented audio samples in 4 seconds. With the "unbatched"
+        suffix, the dataloader will load tfrecords containing unsegmented
+        samples which could be used for learning note sequence in URMP dataset.
+
     """
     self.instrument_key = instrument_key
     self.split = split
     self.base_dir = base_dir
+    self.suffix = suffix if suffix is None else '_' + suffix
```

Comment:
I think `f'{None}'` prints `'None'` here, not an empty string. You probably want to change this line to:

    self.suffix = suffix if '' is None else '_' + suffix